conn.go

```diff
@@ -118,12 +148,12 @@ type conn struct {
 
 	// peer settings
 	peerIdleTimeout  time.Duration // maximum period between sending frames
-	PeerMaxFrameSize uint32        // maximum frame size peer will accept
+	peerMaxFrameSize uint32        // maximum frame size peer will accept
 
 	// conn state
-	errMu sync.Mutex    // mux holds errMu from start until shutdown completes; operations are sequential before mux is started
-	err   error         // error to be returned to client
-	Done  chan struct{} // indicates the connection is done
+	doneErrMu sync.Mutex    // mux holds doneErr from start until shutdown completes; operations are sequential before mux is started
```

Comment:
I like these little renames you've been doing. It really does make it simpler to follow.